components/eamxx/tests/coupled/physics_only/shoc_p3_nudging/input_nudging.yaml

```diff
@@ -14,7 +14,7 @@ atmosphere_processes:
   p3:
     max_total_ni: 720.0e3
   nudging:
-    nudging_filename: [shoc_p3_source_data_${POSTFIX}.INSTANT.nsteps_x${NUM_STEPS}.${RUN_T0}.nc]
+    nudging_filenames_patterns: [shoc_p3_source_data_${POSTFIX}.INSTANT.nsteps_x${NUM_STEPS}.${RUN_T0}.nc]
```

Comment:
One thought, feel free to implement or not: the "patterns" feature is basically not used here, cause there is just one file. What if we generated nudging data with 1 time slice per file? That way, we could specify a pattern, and see if the nudging interface picks up all the files correctly..

Reply:
i dont quite understand this problem.. nudging_filenames_patterns can read {a single filename, a single pattern name, a list of filename, a list of patterns}, so in reading a single filename in this case is just a subset of the ``patterns'' feature ..? (or are you discussing nomenclature?)

oh, the test i added is another one, this is the default input for other nudging tests. 
please see `components/eamxx/tests/coupled/physics_only/shoc_p3_nudging/input_nudging_glob.yaml`

Reply:
I mean, I like the motto "if it's not tested, it doesn't exist". We can certainly to a follow up PR, but I'd like to test the case where we use 2+ files in our nightlies at some point...